### DISCUSSION TITLE
Icon Request: fa-change-case (or fa-all-caps / fa-uppercase)

### DISCUSSION THREAD

**Discussion by timkendrick** [FortAwesome/Font-Awesome#21201] [Icon Requests]
Thanks for Font Awesome, it's an incredible project!

I'm currently working on a side-project that features some WYSIWYG text editor controls, and the only icon I've come across that seems to be missing from the Font Awesome collection is a "Change case" icon like you'd see in MS Word etc.

I'm guessing this is a fairly standard text editor control, so I suppose that would be the main use case – in my case an "All caps" / "Uppercase" icon would actually be even better, since in the app I'm making it's a button that toggles the `text-transform: uppercase` CSS style.

I've attached screenshots of the equivalent icon from MS Word in OS X and Windows if that helps. Thanks!

---

MS Word (Windows):
![screen shot 2015-11-15 at 10 28 52](https://cloud.githubusercontent.com/assets/953720/11168397/44605284-8b85-11e5-8fd4-3d0e09a120f3.png)

MS Word (Mac):
![screen shot 2015-11-15 at 10 37 50](https://cloud.githubusercontent.com/assets/953720/11168399/4cf92420-8b85-11e5-80b8-a95cc6741693.png)

**Comment by mbacon-edocs** [comment]
+1 (bonus would be uppercase, lowercase and sentence case)

**Comment by matthewmorrone** [comment]
bump

**Comment by sindbadi** [comment]
I am looking for similar solution as well.

**Comment by jenstroeger** [comment]
Bump on this: a “[small caps](https://en.wikipedia.org/wiki/Small_caps)” and an “[all caps](https://en.wikipedia.org/wiki/All_caps)” icon would be great!

Here are the icons used by Photoshop:

![Screenshot 2025-12-18 at 06 57 08](https://github.com/user-attachments/assets/7b3c013a-4253-4b8b-bc4a-38e306cd9187)
